app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,7 @@
+class WelcomeController < ApplicationController
+  def index
+    @movies = Movie.order(ranked: :desc).limit(10)
```

Comment:
Because we use `order(ranked: :desc)` here three times, and in each of the model-specific controllers, it might be a good idea to refactor these into class methods (or even better, scopes) on the associated model classes:
```ruby
def Movie < ActiveRecord::Base
  # ...
  def self.by_rank
    return self.order(ranked: :desc)
  end
  # OR...
  scope :by_rank, -> { order(ranked: :desc) }
end
```